### PR TITLE
update urls

### PR DESCRIPTION
**Pull request by AlbRoehm**
- **fix(env-url): update environment urls**
- **fix(demo-ui): add a reset button to be able to jump back to the start**

### CODE REVIEW

**Review by ChaoticJayJay — APPROVED** [review]
Good for me, maybe wait for Vadim review before merged

**Comment by sf-bender** [comment]
### Code Coverage Report
|Overall Project|95.36%|:green_apple:|
|:-|:-|:-:|

|File|Coverage||
|:-|:-|:-:|
|[Hellgate.kt](https://github.com/starfish-codes/hgate2-headless-android-sdk/blob/c4b7fa919e31fae0e654252315aa9c7c85fe6645/sdk%2Fsrc%2Fmain%2Fjava%2Fio%2Fhellgate%2Fandroid%2Fsdk%2FHellgate.kt)|93.99%|:green_apple:|

**Review by lazebny — APPROVED** [review]

**Pull request merged**
